### ISSUE TITLE
Folders are being renamed incorrectly.

### ISSUE THREAD

**Issue by kjac3585** [Radarr/Radarr#2660]
This appears to be a bug that several people have mentioned on Reddit, but with no resolution so far. I managed to get some debug logs of it renaming 2 movies incorrectly. I do not know what is causing it, so reproducing it may be a challenge, it seems almost random as some movie folders are renamed correctly and others are not. IIRC, It's only the folders that are renamed incorrectly, the filenames are good.

**Radarr Version:**
0.2.0.995

**Debug Logs:**
[logs.zip](https://github.com/Radarr/Radarr/files/1833836/logs.zip)

![screencapture-localhost-38083-settings-mediamanagement-2018-03-21-10_10_25](https://user-images.githubusercontent.com/7411003/37718395-96536838-2cf0-11e8-921a-fc94d2bdad44.png)
![screencapture-localhost-38083-movies-national-gallery-266080-2018-03-21-10_10_09](https://user-images.githubusercontent.com/7411003/37718398-97ce7c20-2cf0-11e8-927b-8396e76bffdd.png)
![screencapture-localhost-38083-movies-national-gallery-266080-2018-03-21-10_09_53](https://user-images.githubusercontent.com/7411003/37718403-98d11592-2cf0-11e8-927b-c9a71d6d01b1.png)
![screencapture-localhost-38083-movies-super-high-me-14236-2018-03-21-10_01_57](https://user-images.githubusercontent.com/7411003/37718405-99454f84-2cf0-11e8-9443-8699a870fae9.png)
![screencapture-localhost-38083-movies-super-high-me-14236-2018-03-21-10_01_37](https://user-images.githubusercontent.com/7411003/37718408-99a55e1a-2cf0-11e8-92c7-d3a6fef03b93.png)

**Comment by fryfrog** [comment]
Is there anything different about these two movies? Was one added by hand and one via a list? Or Ombi or something?

**Comment by kjac3585** [comment]
Most of my movies are added manually. I do have a few lists, but I haven't used them much and the only list that auto-imports is "Trakt BoxOffice", which neither of these movies came from. I've had this renaming issue with 20 or 30 movies over the past week or so. These two, in particular, were added manually a while back and then recently were downloaded when I hit "search all" on the wanted page. 

I've also had it rename the folders incorrectly when using "force download". I've yet to figure out what causes it because I can delete one of these erroneously named movies and have it redownloaded and named correctly without ever changing anything other than deleting the movie.

**Comment by galli-leo** [comment]
@kjac3585 By deleting the movie, are you referring to deleting the movie from Radarr or deleting the actual movie file? Radarr doesn't do any renaming of the folder unless you tell it to do so. So it will always have the path, which you give it at the beginning.

**Comment by kjac3585** [comment]
Why was this closed? It seems like a potential bug to me.

I mean deleting the entire movie from Radarr and my harddrive and then re-adding the movie to Radarr. Sometimes it will download the movie and rename it correctly, other times it doesn't.

**Comment by galli-leo** [comment]
Because according to the debug logs, there isn't any reason why this is a bug. How do you exactly readd the movies to Radarr?

**Comment by kjac3585** [comment]
The re-adding part is just me trying to narrow down why the folders are being named wrong. but I hit the delete button in Radarr and hit YES on delete all files. Then I search for the movie again and add it back.

The reason I thought it was a bug is that there is no existing folder for (for example) "National.Gallery.2014.LiMiTED.DVDRip.x264-TASTE" in my L:\_MOVIES directory (import dir) and the directory "L:\_MOVIES\National.Gallery.2014.LiMiTED.DVDRip.x264-TASTE" does not exist prior to Radarr creating it when it imported the movie. Since I have Radarr setup to rename folders to Moviename (year) and it isn't doing that for some movies, I assumed it was a bug. Could there be something else interfering with the renaming process? I've been running Radarr for a few months now and this problem started maybe a week or two ago.

**Comment by galli-leo** [comment]
@kjac3585 When you search for the movie again, you do this via the add new movie, where you get a search bar?

**Comment by fryfrog** [comment]
I actually agree with @kjac3585, this feels like a bug that hasn't been figured out. And it has been a fairly common report over the past week or so on /r/radarr. Nothing I've seen points at a user issue, though I also can't see where it went wrong on Radarr side.

**Comment by kjac3585** [comment]
That is correct. I can delete/re-add again and send the debug logs if it makes any difference, but with no errors being reported I don't know what you'd gain from that.

**Comment by fryfrog** [comment]
Right, but does the delete / re-add fail again w/ the same rename problem or does it work correctly?

**Comment by kjac3585** [comment]
@fryfrog I was replying to @galli-leo on that last comment, but IIRC sometimes it does and sometimes it doesn't. I'll be able to test it again in 30 minutes or so and I can post the results.

**Comment by kjac3585** [comment]
Ok, so I've learned a little bit more now. The Super High Me movie already existed in the correct folder, but Radarr downloaded it again for some reason and imported it to a different directory. The National Gallery movie was deleted and re-added, it downloaded and was imported to the correct directory this time. The confusing part is I looked at another movie and noticed the directory shown in Radarr was set incorrectly from the very beginning. If I look at the movie's history tab it shows that it has never been downloaded and the movie and folder do not exist on my hdd either. I'm not sure where or why it has "L:\_MOVIES\The.Institute.2013.720p.WEB-DL.DD5.1.H.264-wndk" as the default directory.

![screencapture-localhost-38083-movies-the-institute-159988-2018-03-21-14_21_47](https://user-images.githubusercontent.com/7411003/37732751-f6a7e7d6-2d13-11e8-930e-5f48e86f99e6.png)

**Comment by kjac3585** [comment]
Alright, I think I figured it out now and it looks like "user error". A while back I grabbed a text list of movies in typical "scene text format" and a little script "text2folders" in order to import them into Radarr. At the time I thought Radarr would simply rename the folders correctly, but I guess that's not the case. I'm still not sure why it downloaded "super high me" twice though, that movie wasn't in my list.
